build.py

```diff
@@ -38,8 +38,8 @@ def get_rvm_path():
     cmd = ['whereis', 'rvm']
     p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     _out, _err = p.communicate()
-    index = len(_out.lstrip().split(': ')) - 1
-    rvm_path = _out.lstrip().split(': ')[index]
+    index = len(_out.lstrip().split(': ')) - 1      # b': ' for Python 3
+    rvm_path = _out.lstrip().split(': ')[index]     # b': ' for Python 3
```

Comment:
These comments can be removed. The change that are talking about is only necessary if the python scripts are run using Python 3.

Reply:
fine to leave here for now - we'll get to py3 soon enough

Reply:
Leaving as-is.